travis test-rendering failures

The test-rendering are failing on travis. 
This is probable related to the fact that the latest version of firefox is used:
https://github.com/openlayers/openlayers/blob/master/.travis.yml#L8-L9

With firefox 53, I've the following error:
```
$ make test-rendering 
node tasks/test-rendering.js
Gecko error: it seems /tmp/firefox/firefox is not compatible with SlimerJS.
See Gecko version compatibility. If version is correct, launch slimerjs
with --debug=true to see Firefox error message
Makefile:146: recipe for target 'test-rendering' failed
make: *** [test-rendering] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ node_js:
   - "6.1"
 
 addons:
-  firefox: "latest"
+  firefox: "52.0"
 
 cache:
   directories:

